acra-censor/common/logging_logic.go

```diff
@@ -134,13 +127,13 @@ func (queryWriter *QueryWriter) Release() {
 }
 
 func (queryWriter *QueryWriter) reset() {
+	queryWriter.mutex.Lock()
+	defer queryWriter.mutex.Unlock()
```

Comment:
`defer` good to use if we have a lot of points of exit to avoid multiple calls of `Unlock`. For example
```
func t(){
   l.Lock()
   _, err := func1()
   if err != nil{
      l.Unlock()
      return
   }
   _, err := func2()
  if err != nil{
      l.Unlock()
      return
   }
   _, err := func3()
  if err != nil{
      l.Unlock()
      return
   }
}
```
In such case to not forget unlock and call it once we should use `defer l.Unlock` and it simple
But `defer` has a small overhead. It's really small but it have. And when we have simple cases such this we can avoid it without any care that we can forget